src/test/java/me/karakelley/http/ResponseFormatterTest.java

```diff
@@ -1,22 +1,27 @@
 package me.karakelley.http;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class ResponseFormatterTest {
+  Response response;
+
+  @BeforeEach
+  void setUp() {
+    response = new Response();
```

Comment:
Any reason the `setStatus("200 OK")` isn't here as well? I'm on the fence about it myself...